### ISSUE TITLE
boot-swaggerSwagger docs endpoint: (http://localhost:8080/springfox/v2/api-docs) gives a 404

### ISSUE THREAD

**Issue by marceloverdijk** [springfox/springfox-demos#20]
`boot-swagger` demo's Swagger docs endpoint: (http://localhost:8080/springfox/v2/api-docs) gives a `404`.

Other urls:
* All Swagger Resources(groups) http://localhost:8080/springfox/swagger-resources
* Swagger UI endpoint: http://localhost:8080/springfox/swagger-ui.html
do give back expected results.

I found this out as I have the same problem in my own app.
Note that `boot-static-docs`s Swagger docs endpoint does work.

**Comment by marceloverdijk** [comment]
I think the url in the docs is not valid as it should contain the group like:
`http://localhost:8080/springfox/v2/api-docs?group=full-petstore-api

**Comment by dilipkrish** [comment]
yes.. this project needs some love. Thanks for pointing that out
